Source/VisualStyle.swift

```diff
@@ -105,7 +110,7 @@ extension VisualStyle {
 public class DefaultVisualStyle: VisualStyle {
 
     private let alertStyle: AlertControllerStyle
-    init(alertStyle: AlertControllerStyle) { self.alertStyle = alertStyle }
+    public init(alertStyle: AlertControllerStyle) { self.alertStyle = alertStyle }
```

Comment:
I don't follow. Doesn't a subclass simply inherit the init method which you can then use to instantiate the subclass?